content/elections.md

```diff
@@ -11,13 +11,10 @@ dek = "Candidates for president, U.S. House and Senate, row offices, and more wi
 extended-kicker = "Pa. Election Center 2024"
 linktitle = "Everything you need to prepare for Pa.’s 2024 election"
 blurb = "Spotlight PA wants to empower voters to make an informed decision on May 16. Our new Election Center features key dates, a campaign finance tracker, in-depth coverage, and more."
-url = "/elections-2024/"
+url = "/elections/"
 aliases = [
     "/election/",
     "/elections/",
-    "/elections-2023/",
```

Comment:
I guess retain the 2023 alias, since it doesn't have an old page anymore. I think at some point, Jeff had a standalone 2023 page setup for awards, but it got pulled out when 2024 happened.